generator/webapp/src/main/java/io/fabric8/maven/generator/webapp/WebAppGenerator.java

```diff
@@ -130,6 +134,11 @@ private AppServerHandler createCustomAppServerHandler(String from) {
     }
 
     private AssemblyConfiguration createAssembly(AppServerHandler handler) {
+        String path = getConfig(Config.path);
+        if (path.equals("/")) {
+            path = "ROOT";
```

Comment:
Question, Just to help me understand for this fix. So we're validating <finalName> here, right? final name of jar would default to ROOT.war , am I right?